docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -22,6 +22,12 @@ This is a list of things to check if you want to update from a previous version
 
 ## Migration from 2022.01.00 to 2022.02.00
 
+### MailingLists plugin has been removed
+
+This plugin has ben removed from the core of mapstore and if you want to include it again in you projects
+you have to include it as a custom plugin.
+
```

Comment:
This goes in a section `Migration from 2022.01.00 to  2022.01.01`.
Not needed to specify how to restore it. 

```suggestion
### MailingLists plugin has been removed

`MailingLists` plugin has ben removed from the core of MapStore. This means you can remove it from your `localConfig.json` (if present, it will be anyway ignored by the plugin system).

```